libtailscale/net.go

```diff
@@ -239,6 +240,7 @@ func (b *backend) NetworkChanged() {
 		if nm, ok := b.sys.NetMon.GetOK(); ok {
 			nm.InjectEvent()
```

Comment:
per the docs you just wrote in the oss CL, you want to set the atomic first and THEN call this InjectEvent, so your line 243 should be up at 240, no?